CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ### v1.0.0 @ 2020-09-30
-* [2020-09-20 10:47:15 CDT] - Initial.
+* [2020-09-20 10:47:15 CDT] - Initial:
+    * [ls-by-min] Sort `ls` by file size, descending.
+    * [stream-to-youtube.sh] Live Screencast directly to YouTube from the CLI (via ffmpeg).
```

Comment:
- [ ] stream-to-youtube needs to be renamed from stream_to_youtube.sh
- [ ] stream-to-youtube needs a README.md entry...